### ISSUE TITLE
Add backup tokens

### ISSUE THREAD

**Issue by 9mido** [fallenhitokiri/otp-demo#1]
It would be nice to have some backup tokens generated in case the user loses their OTP device and be able to use those backup tokens to login to admin and setup a new device for that user or change previous configuration.

**Comment by fallenhitokiri** [comment]
I agree, this should exist in a production ready library. As backup tokens need a little bit care to be implemented properly I skipped them for the purpose of the demo and blog post. If I decide to make this a full lib I will definitely include this feature - more on this in #2 - keeping the issue open as reference.
